### PR TITLE
adding CONTRIBUTING.md

### LINKED ISSUE #15: [FEATURE REQUEST] [DOC] Add CONTRIBUTING.md
**Is your feature request related to a problem? Please describe.**
No guide is available for new contributors.

**Describe the solution you'd like**
Write a clear and concise `CONTRIBUTING.md` for new contributors.
It should include the following points: 
- Read all the other `.md` files such as `README.md`, `CODE_OF_CONDUCT.md`, etc. before contributing.
- Add step by step instructions to fork, clone, create a branch, make changes, push to fork and open a PR.
- Add the tech stack of the project in the description: NextJS + TypeScript
- Add all other necessary things in the `CONTRIBUTING.md` file (try to read other projects' `CONTRIBUTING.md` to see what else is required

### PR DESCRIPTION
**Pull request by An-Array**
closes #15 
Is there anything else to add or to change

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+- Maintain Code Quality: Follow the project's coding guidelines and best practices.
+- Test Your Code: Ensure that your contributions are thoroughly tested and do not break existing functionality.
+- Documentation: Clearly document your code, changes, and additions.
+- Respect Licensing: Respect and adhere to the project's licensing terms.
+
+### Code of Conduct
+
+Please adhere to our [Code of Conduct](CODE_OF_CONDUCT.md) when participating in Timeloom. We encourage a respectful and inclusive environment for all contributors.
+
+## Submitting Your Contributions
```
Comment (inclinedadarsh) [inline CONTRIBUTING.md:68]: Can you list every step clearly, with screenshots and bash commands whenever possible.

It's for beginners, so it'll help them a ton

**Review by inclinedadarsh — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (An-Array) [reply CONTRIBUTING.md:68]: Sure mate

**Comment by An-Array** [comment]
This works? @inclinedadarsh

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -67,22 +67,87 @@ Please adhere to our [Code of Conduct](CODE_OF_CONDUCT.md) when participating in
...
 
 ## Additional Information
 
 For additional questions or support, you can:
 
 Visit the [Hacktoberfest official website](https://hacktoberfest.com) for event details.
 Reach out to project maintainers for specific project-related inquiries.
-Thank you for contributing to Hacktoberfest 2023! Happy coding! 🖥️
\ No newline at end of file
+Thank you for contributing to Hacktoberfest 2023! Happy coding! 🖥️
```
Comment (inclinedadarsh) [inline CONTRIBUTING.md]: Everything looks great.

Just add my github https://github.com/inclinedadarsh, email dubeyadarshmain@gmail.com and [twitter](https://twitter.com/inclinedadarsh) in the support section.

**Review by inclinedadarsh — CHANGES_REQUESTED** [review]

**Review by inclinedadarsh — APPROVED** [review]

**Comment by inclinedadarsh** [comment]
Thanks for your contribution @An-Array 

It would be great if you could leave a star on this repository.
Thanks and regards!

**Pull request merged**